.travis.yml

```diff
@@ -1,14 +1,22 @@
 language: java
 sudo: false
+
 cache:
   directories:
   - .autoconf
   - $HOME/.m2
+  - $HOME/.zinc
+
+script: mvn scoverage:report
```

Comment:
OK, I played with this locally and can only get a non-empty coverage report when the tests are run as part of `scoverage:report`.

Things that didn't work:

```
mvn clean -Pcoverage test  # also without -Pcoverage
mvn -Pcoverage -DskipTests scoverage:report
mvn -Pcoverage coveralls:report
```

So I think we should leave it like I have it here; I've added comments to `.travis.yml` about this state of affairs though.